### PR TITLE
feat: add dismissable, text and shadow card

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds three new card variants to the component library.

- Dismissible card
- Card with only text and
- Card with shadow

image
image
image

### CODE REVIEW

**Comment by shrushti2000** [comment]
Add javascript code snippet in the documentation under dismissable cards so that the user can copy js part as well. Also you can add a reset button for dismissable  card.
The UI for card looks clean. 
Great job.

File: docs.html [hunk 1]
```diff
@@ -218,6 +218,75 @@ <h6 class="card-title">Macbook Air M2, 32GB</h6>
...
+                            <button class="card-btn card-btn-primary">Add to Cart</button>
+                            <button class="card-btn card-btn-secondary">Save For Later</button>
+                            <i class="card-btn-icon far fa-heart"></i>
+                            <i class="card-btn-icon fas fa-ellipsis-v"></i>
+                        </div>
+                        <div class="card-badge">-22%</div>
+                        <i class="card-dismiss-icon fas fa-times"></i>
+                    </div>
+
+                    <!--
```
Comment (Shailseen) [inline docs.html:239]:
```sugg
```

Comment (Shailseen) [inline docs.html:239]: Can you remove this comment? and give this code on the UI . So usEr can use this.

**Review by Shailseen — APPROVED** [review]
Only one changes you have to do ....code written nicely.

File: docs.html (hunk 1 above)
Comment (vishalg8454) [reply docs.html:239]: okay

**Pull request merged**